Client/Frontend/Browser/DomainUserScript.swift

```diff
@@ -58,6 +59,8 @@ enum DomainUserScript: CaseIterable {
                  "talk.bravesoftware.com", "beta.talk.bravesoftware.com",
                  "dev.talk.brave.software", "beta.talk.brave.software",
                  "talk.brave.software"])
+    case .bravePlaylistFolderSharingHelper:
+      return Set(["playlist.bravesoftware.com", "playlist.brave.com", "127.0.0.1", "brandon-t.github.io"])
```

Comment:
With the endpoint being live, we can remove test endpoints now

Reply:
Fixed